docs/whats-new/2020-08.md

```diff
@@ -0,0 +1,142 @@
+- [Cancel a list of tasks (C#)](/dotnet/csharp/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks) - Update async concept articles in the C# programming guide
+- [Process asynchronous tasks as they complete (C#)](/dotnet/csharp/programming-guide/concepts/async/start-multiple-async-tasks-and-process-them-as-they-complete) - Update async concept articles in the C# programming guide
+- [Asynchronous file access (C#)](/dotnet/csharp/programming-guide/concepts/async/using-async-for-file-access) - Update async concept articles in the C# programming guide
+- [A tour of the C# language](/dotnet/csharp/tour-of-csharp/index) - Restructure and modernize tour of C#
+
+## .NET Framework
+
+### New articles
+
+- [ControlBuilderInterceptor class](/dotnet/framework/additional-apis/controlbuilderinterceptor-class) - Add ControlBuilderInterceptor to additional APIs
+
+## F# language
+
+### New articles
+
+- [Plain text formatting](/dotnet/fsharp/language-reference/plaintext-formatting) - F# plaintext printing documentation
+
+### Updated articles
+
+- [F# Interactive options](/dotnet/fsharp/language-reference/fsharp-interactive-options) - F# plaintext printing documentation
+
+## .NET fundamentals
+
+### New articles
+
+- [Overview of .NET source code analysis](/dotnet/fundamentals/productivity/code-analysis) - Initial port of code analysis docs
+- [Configure code analysis rules](/dotnet/fundamentals/productivity/configure-code-analysis-rules) - Initial port of code analysis docs
+
+## ML.NET
+
+### New articles
+
+- [How to install GPU support in Model Builder](/dotnet/machine-learning/how-to-guides/install-gpu-model-builder) - Add Model Builder GPU instructions
+
+## .NET
+
+### Updated articles
+
+- [Custom date and time format strings](/dotnet/standard/base-types/custom-date-and-time-format-strings) - Update headings, minor clean up, and add correct ms.topic
+
+## Community contributors
+
+The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).
+
+- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (49)
+- [NextTurn](https://github.com/NextTurn) - Next Turn (20)
+- [JohyPark](https://github.com/JohyPark) - Joohyung Park (6)
+- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (6)
+- [sughosneo](https://github.com/sughosneo) - Sumit Ghosh (6)
+- [bradriske](https://github.com/bradriske) (2)
+- [Mrlinkwii](https://github.com/Mrlinkwii) (2)
+- [AbregoDev](https://github.com/AbregoDev) - Daniel Moreno (1)
+- [AChehre](https://github.com/AChehre) - Ahmad Chehreghani (1)
+- [akashmanoj213](https://github.com/akashmanoj213) - Akash Manoj (1)
+- [anadem](https://github.com/anadem) - Alan Sinclair (1)
+- [angularsen](https://github.com/angularsen) - Andreas Gullberg Larsen (1)
+- [azure-sdk](https://github.com/azure-sdk) - Azure SDK Bot (1)
```

Comment:
this in theory is probably an employee behind a bot?

Reply:
thanks. I have made a code change to fix this, but we haven't deployed it yet.